examples/api/lib/material/ink/ink.image_clip.0.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for Image.frameBuilder
+
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(const MyApp(
+    image: NetworkImage(
+      'https://flutter.github.io/assets-for-api-docs/assets/widgets/puffin.jpg',
+    ),
+  ));
+}
+
+class MyApp extends StatelessWidget {
```

Comment:
For clarity, this example could be simplified a little bit by moving the stuff from this widget's build method directly into runApp above. That moved the example's focus to the important MyStatelessWidget.